hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieAppendHandle.java

```diff
@@ -444,7 +448,9 @@ private void writeToBuffer(HoodieRecord<T> record) {
     }
     Option<IndexedRecord> indexedRecord = getIndexedRecord(record);
     if (indexedRecord.isPresent()) {
-      recordList.add(indexedRecord.get());
+      if (indexedRecord.get() != IGNORE_RECORD) { // Skip the Ignore Record.
```

Comment:
move the annotation to upper line

Reply:
> @pengzhiwei2018 one more question, will we introduce Catalog to manage table operations in further?

Yes, I agree with introduce Catalog to manage table operations for spark3 in the further.

Reply:
Fixed~